chapter-15/recipe-03/cxx-example/dashboard.cmake

```diff
@@ -0,0 +1,29 @@
+set(CTEST_PROJECT_NAME "example")
+set(CTEST_SITE "localhost")
+set(CTEST_BUILD_NAME "${CMAKE_SYSTEM_NAME}-${CMAKE_HOST_SYSTEM_PROCESSOR}")
+
+set(CTEST_SOURCE_DIRECTORY "${CTEST_SCRIPT_DIRECTORY}")
+set(CTEST_BINARY_DIRECTORY "${CTEST_SCRIPT_DIRECTORY}/build")
+set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
```

Comment:
Is this necessary? It might make trouble on Windows...

Reply:
The generator has to be given but the alternative is this:
```
ctest -S dashboard.cmake -D CTEST_CMAKE_GENERATOR="Unix Makefiles"
```
Should we go for that? Anyway running the CDash script is currently not part of the automated testing.

Reply:
Comment in source + note in TypeCloud are enough for me ;)

Reply:
OK. I agree. Good compromise.